spec/models/user_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'validations' do
+    subject { build(:user) }
+
+    it { is_expected.to validate_presence_of(:email) }
+    # Idk why this isn´t working
```

Comment:
Did you try something like [this](https://github.com/thoughtbot/shoulda-matchers/issues/1092#issuecomment-378864653)?

Reply:
Thank you so much for that suggestion! <3